docs/customization/advanced.md

```diff
@@ -221,3 +221,17 @@ module.exports = neutrino => {
 Presets can also have their own custom data in addition to the Neutrino config. See your respective preset for details.
 Again, rather than reiterate the documentation for [webpack-chain](https://github.com/mozilla-neutrino/webpack-chain)
 here, please refer to its documentation for all ways you can modify a config instance to solve your use cases.
+
+### Conditional configuration
+
+Some plugins and rules are only available in certain environments. For example, the Web preset only exposes a minify
+plugin during production, leading to issues when trying to modify its settings, but blows up during development.
```

Comment:
Switch "blows up" to "throws an exception". I typically avoid using colloquial prose in formal documentation.